### ISSUE TITLE
EPIC: MediaCurrent AAA Accessibility Recommendations

### ISSUE THREAD

**Issue by dneufeldcu** [cedardevs/onestop#876]
### Description
AAA findings (optional - complex)

### Tasks
- [ ] provide ability to change color themes with custom color picker (predefined colors may not work for all users)
- [ ] update button instead of auto refresh for filters
  - unexpected context changes can be difficult for some types of users. some alternative options:
  - [ ] ability to disable auto refresh
  - [ ] more verbose announcements when content updates
- [ ] indicators of location (context and navigation)
  - improve browsability of data available
  - indicate popular topics are search terms
  - categorize content
  - breadcrumb trail
  - human navigation sitemap
- [ ] Preview images without alt text (collection detail view)
  - indicate image is decorative in alt text
  - provide feedback to data managers when alt text is missing (analyzer in IM?)
  - provide clear information about the accessibility of linked data files
- [ ] define unusual words, abbreviations
  - link to definition lists
  - provide glossary
  - inline definitions using `<dfn>`
  - wikipedia style tooltip
- [ ] adjust reading level (analyzer feedback to data managers?)

**Comment by dneufeldcu** [comment]
OBE no expressed interest in AAA support at this time.
